src/services/imageBackend.ts

```diff
@@ -1,21 +1,28 @@
+import { testImages } from '../testData';
```

Comment:
This import does not work due to your updated .gitignore hence the failing Nelify build / deploy